src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
```

Comment:
The variable `$font-size` is used but not defined in this file. Make sure it is defined in this file or imported from another SCSS file.